src/common/service/NetRemoteService.cxx

```diff
@@ -389,6 +424,156 @@ NetRemoteService::WifiAccessPointDisable([[maybe_unused]] ::grpc::ServerContext*
     return grpc::Status::OK;
 }
 
+using Microsoft::Net::Wifi::Dot11PhyType;
+
+::grpc::Status
+NetRemoteService::WifiAccessPointSetPhyType([[maybe_unused]] ::grpc::ServerContext* context, const ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeRequest* request, ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeResult* response)
+{
+    using Microsoft::Net::Wifi::Ieee80211AccessPointCapabilities;
+
+    LOGD << std::format("Received WifiAccessPointSetPhyType request for access point id {}", request->accesspointid());
+
+    WifiAccessPointOperationStatus status{};
+    status.set_code(WifiAccessPointOperationStatusCode::WifiAccessPointOperationStatusCodeSucceeded);
+
+    auto handleFailure = [&](WifiAccessPointOperationStatusCode statusCode, const std::string& statusMessage) {

```

Comment:
nit: It makes sense to generally prefer `std::string_view` over `const std::string&` when a non-owning, immutable copy is being passed (eg. read-only scenarios) as `std::string_view` is lighter weight than the reference, and conveys non-ownership by definition. 

Here a copy is being made, so it probably makes even more sense to just use `std::string` instead, then the caller is 100% aware that a copy is being made and this functions owns that copy and can do whatever it wants with it. 

```suggestion
    auto handleFailure = [&](WifiAccessPointOperationStatusCode statusCode, std::string statusMessage) {
```

Reply:
Oh never mind, it seems like `status.set_message` makes a copy